src/docs/product/integrations/integration-platform/index.mdx

```diff
@@ -176,17 +199,29 @@ def refresh_token(install_id):
     return new_token
 ```
 
-Instead of keeping track of times and passively refreshing at the time a token expires, one painless way you can handle refreshing tokens is to actively capture exceptions raised by requests that receive a 401 Unauthorized response from Sentry, refresh the token, and remake the request.
+Instead of keeping track of times and passively refreshing when a token expires, one painless way you can handle refreshing tokens is to actively capture exceptions raised by requests that receive a 401 unauthorized response from Sentry, refresh the token, and remake the request.
```

Comment:
We want to avoid saying how a process is going to feel for a user. 

```suggestion
Instead of keeping track of times and passively refreshing when a token expires, another way you can handle refreshing tokens is to actively capture exceptions raised by requests that receive a 401 unauthorized response from Sentry, refresh the token, and remake the request.
```